### DISCUSSION TITLE
Suggestion: Enable Signatures in the forum

### DISCUSSION THREAD

**Discussion by eldexterr** [RetroAchievements/RAWeb#773] [Ideas]
A very common function in a forum, why not? :+1:

**Comment by luchaos** [comment]
It's a feature older forum software have but I've personally not been a fan of. Signatures tend to take up a lot of space. 
In our case there's already a tooltip showing user details including the user motto. Maybe we should think about extending the user motto capabilities instead?

**Comment by eldexterr** [comment]
Yeah is kind of an outdated feature now days, but was just a suggestion isn't a very bid deal if isn't implemented :)

**Comment by jeffjenx** [comment]
Forum signatures were notoriously abused by users to post links back to their website in order to boost SEO performance. I agree with @luchaos that the only purpose they serve is to clutter up the communication stream.

Personally, I like the current motto model as it prevents misuse through restrictions, similar to Xbox's gamer profile. If anything, maybe a WYSIWYG editor for a more broad profile description would suffice (a la Stack Overflow) and some related social link fields, but only display it on the user's profile page.

Not sure if that is planned for 2.x release.
